.travis.yml

```diff
@@ -13,4 +13,5 @@ script:
   - pre-commit run -a
   - npm install -g expo-cli --loglevel=error
   - npm install --log-level=error
+  - npm test -- -u
```

Comment:
For Travis, the `-u` flag to update the snapshot shouldn't be here. It should only be used by devs
Travis should match whatever is pushed, and not create its own snapshot baseline